sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -58,6 +58,17 @@ private Sentry() {}
     return hub;
   }
 
+  /**
+   * Returns a new hub which is cloned from the mainHub.
+   *
+   * @return the hub
+   */
+  @ApiStatus.Internal
+  @ApiStatus.Experimental
+  public static @NotNull IHub getNewHub() {
```

Comment:
maybe naming it something like `cloneMainHub` or `mainHubClone` is better? Because `getNewHub` is not self-descriptive (e.g. from the first look it's not clear what's the diff between getCurrentHub and getNewHub)